frontend/src/app/modules/grids/widgets/news/news.component.html

```diff
@@ -1,40 +1,34 @@
-<widget-header
-    [name]="widgetName"
-    [editable]="isEditable">
+<widget-header [name]="widgetName" [editable]="isEditable">
 
-  <widget-menu
-      [resource]="resource">
+  <widget-menu [resource]="resource">
   </widget-menu>
+  
 </widget-header>
-
 <div class="grid--widget-content">
-  <no-results *ngIf="noEntries"
-             [title]="text.noResults">
+  <no-results *ngIf="noEntries" [title]="text.noResults">
   </no-results>
   <ul class="widget-box--arrow-links">
     <li class="-widget-box--arrow-multiline" *ngFor="let news of entries">
       <div>
-        <user-avatar *ngIf="news.author"
-                     [user]="news.author"
-                     data-class-list="avatar avatar-medium -spaced">
+        <user-avatar *ngIf="news.author" [user]="news.author" data-class-list="avatar avatar-medium -spaced">
         </user-avatar>
-        <a [href]="newsProjectPath(news)"
-           [textContent]="newsProjectName(news)">
+        <a [href]="newsProjectPath(news)" [textContent]="newsProjectName(news)">
         </a>:
-        <a [href]="newsPath(news)"
-           [textContent]="news.title">
+        <a [href]="newsPath(news)" [textContent]="news.title">
         </a>
 
-        <p *ngIf="news.author"
+        <p *ngIf="news.summary" [innerText]="news.summary"></p>
```

Comment:
"innerText" property is non-standard. Prefer to use `[textContent]="news.summary"`